ansible/group_vars/all/piecewise_global_config.yml

```diff
@@ -0,0 +1,278 @@
+#######
+# PIECEWISE SYSTEM CONFIGURATION
+#######
+
+# Certain tasks may operate in a less secure (but more convenient) manner, eg.
+# enabling passwordless sudo or generating self-signed ssl certs, when testing
+# locally, in Vagrant. But not in production!
+#
+# Supported values: production, development
+env: development
+
+# What Linux distro is being used? Supported options include: debian8, rhel7
+#sys: debian8
+
+# The project name should be the name of your forked repository.
+# This var is referenced by a few other vars, eg. git_repo, hostname, site_fqdn.
+project_name: piecewise
+
+# Information about the fork and branch to deploy.
+# Specify any valid git remote (typically a github username)
+remote: https://github.com/critzo
```

Comment:
Is there a way to do this as a variable like how we do on the m-lab site? or will this require a manual change?